.github/workflows/publish-extension-nightly.yml

```diff
@@ -79,7 +79,16 @@ jobs:
         env:
           VSCE_PAT: ${{ secrets.VSCODE_PUBLISH_PWD }}
         working-directory: ./packages/plugin-core
-        run: yarn deploy:vscode
+        run: |
+          npx verdaccio > verdaccio.log  2>&1 &
+          sleep 30
```

Comment:
why sleep 30? usually completes in <10s

Reply:
I was just testing with higher numbers in the hope of reducing noise and getting the pipeline unblocked, but this can be reduced.